sys/net/gnrc/network_layer/sixlowpan/frag/rbuf.c

```diff
@@ -29,8 +29,15 @@
 #define ENABLE_DEBUG    (0)
 #include "debug.h"
 
+/* estimated fragment payload size to determinate RBUF_INT_SIZE, default to
+ * frame size - fragment header. */
+#ifndef GNRC_SIXLOWPAN_FRAG_SIZE
+#define GNRC_SIXLOWPAN_FRAG_SIZE (102 - 5)
+#endif
+
 #ifndef RBUF_INT_SIZE
-#define RBUF_INT_SIZE   (GNRC_IPV6_NETIF_DEFAULT_MTU * RBUF_SIZE / 127)
+#define RBUF_INT_SIZE ((GNRC_IPV6_NETIF_DEFAULT_MTU + GNRC_SIXLOWPAN_FRAG_SIZE - 1) / \
+                       GNRC_SIXLOWPAN_FRAG_SIZE * RBUF_SIZE)
```

Comment:
Could you add some parenthesis please? I'm getting confused about the parsing ^^